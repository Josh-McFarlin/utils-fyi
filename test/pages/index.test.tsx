import React from "react";
import { render } from "../testUtils";
import HomePage from "../../src/pages/index";

describe("Home Page", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<HomePage />, {});

    expect(asFragment()).toMatchSnapshot();
  });
});
