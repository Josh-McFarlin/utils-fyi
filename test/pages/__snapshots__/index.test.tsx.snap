// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page matches snapshot 1`] = `
<DocumentFragment>
  <h1>
    Welcome!
  </h1>
  <h3>
    This website is coming soon
  </h3>
</DocumentFragment>
`;
